### DISCUSSION TITLE
How to set secret key in vercel environment variables?

### DISCUSSION THREAD

**Discussion by Squirrelcoding** [vercel/vercel#7580] [Help]
I am trying to deploy my application, and added all the environment variables but of course, I got an error. Just what I needed. I only have a simple question and that is how do *you* guys set secret keys? Do you wrap it in single quotes or something? I only need an example to follow!

**Comment by raul-saez** [comment]
I think you need to be more specific on the steps you're taking and what the error is you're getting. The [docs](https://vercel.com/docs/concepts/projects/environment-variables) are pretty clear on this. Once you've set your environment variables on your Vercel project you should simply be able to access them with `process.env.<ENV_NAME>`.

**Comment by SoniaBisinger** [comment]
for DJANGO:
settings.py : 
SECRET_KEY = os.environ.get('SECRET_KEY')

on your project on Vercel -> settings -> Environment variables, add
Name: SECRET_KEY
Value: your_password
No double/single quotes needed
